doc/classes/ProjectSettings.xml

```diff
@@ -92,9 +92,11 @@
 			</argument>
 			<argument index="1" name="replace_files" type="bool" default="true">
 			</argument>
+			<argument index="2" name="offset" type="int" default="0">
+			</argument>
 			<description>
 				Loads the contents of the .pck or .zip file specified by [code]pack[/code] into the resource filesystem ([code]res://[/code]). Returns [code]true[/code] on success.
-				[b]Note:[/b] If a file from [code]pack[/code] shares the same path as a file already in the resource filesystem, any attempts to load that file will use the file from [code]pack[/code] unless [code]replace_files[/code] is set to [code]false[/code].
+				[b]Note:[/b] Load with offset feature only supports .pck file. If a file from [code]pack[/code] shares the same path as a file already in the resource filesystem, any attempts to load that file will use the file from [code]pack[/code] unless [code]replace_files[/code] is set to [code]false[/code].
```

Comment:
The previous note is unrelated, so instead of putting both on the same paragraph, they should be separate. The docs should also explain what the offset actually is and means.

```suggestion
				The optional [code]offset[/code] parameter can be used to ... (what can it be used for in the context of a PCK?). It is only supported for .pck files.
				[b]Note:[/b] If a file from [code]pack[/code] shares the same path as a file already in the resource filesystem, any attempts to load that file will use the file from [code]pack[/code] unless [code]replace_files[/code] is set to [code]false[/code].
```

Reply:
Got it, will work on that!

Reply:
Or alternatively it could be a second `[b]Note:[/b]` coming after the first one, to keep the same order as the arguments.

Reply:
Thanks for the suggestions! I updated the documentation accordingly. (Also I just did a push a seconds ago and removed an extra whitespace that is causing format checker to fail)